build-version.sh

```diff
@@ -38,10 +38,24 @@ elif [ "$TARGET_ARCH" == "aarch64-unknown-linux-gnu" ]; then
     rustup target add "$TARGET_ARCH"
     if ! [ -f "$HOME/.cargo/config" ]; then
         echo "[target.aarch64-unknown-linux-gnu]" >>~/.cargo/config
-        echo "linker = \"$PWD/zig-aarch64.sh\"" >>~/.cargo/config
+        echo "linker = \"$PWD/zig-aarch64-gnu.sh\"" >>~/.cargo/config
     fi
 
-    CARGO_PROFILE_RELEASE_CODEGEN_UNITS="1" CARGO_PROFILE_RELEASE_LTO="fat" OPENSSL_STATIC=1 CC=./zig-aarch64.sh cargo install "$CRATE" --version "$VERSION" --target "$TARGET_ARCH"
+    CARGO_PROFILE_RELEASE_CODEGEN_UNITS="1" CARGO_PROFILE_RELEASE_LTO="fat" OPENSSL_STATIC=1 CC=./zig-aarch64-gnu.sh cargo auditable install "$CRATE" --version "$VERSION" --target "$TARGET_ARCH"
```

Comment:
You need to

```suggestion
    rm -f ~/.cargo/.crates.toml ~/.cargo/.crates2.json 
    CARGO_PROFILE_RELEASE_CODEGEN_UNITS="1" CARGO_PROFILE_RELEASE_LTO="fat" OPENSSL_STATIC=1 CC=./zig-aarch64-gnu.sh cargo auditable install "$CRATE" --version "$VERSION" --target "$TARGET_ARCH"
```

before running `cargo-build`.